src/csv.c

```diff
@@ -113,7 +105,31 @@ static int value_list_to_filename(char *buffer, size_t buffer_size,
     ptr += len;
   }
 
-  status = FORMAT_VL(ptr, ptr_size, vl);
+  char *plugin = NULL;
+  char *plugin_instance = NULL;
+  char *type = NULL;
+  char *type_instance = NULL;
+
+  for (size_t j = 0; j < m->label.num; ++j) {
+    label_pair_t *l = m->label.ptr + j;
+
+    if (plugin == NULL) {
+      plugin = strdup(l->name);
+      plugin_instance = strdup(l->value);
+    } else if (type == NULL) {
+      type = strdup(l->name);
+      type_instance = strdup(l->value);
+    }
+  }
+
+  status = format_name(ptr, ptr_size, NULL, plugin, plugin_instance, type,
```

Comment:
`format_name()` segfaults when given NULL hostname.  Change it e.g. to a fixed string (say ""), to get the plugin working.